azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -65,8 +73,13 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 				Optional: true,
 			},
 
+			"enable_automatic_failover": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
since this'll be `false` if it's not specified, should we default this to a value or make it required?

Reply:
When created via the portal its not required and seems to default to `false`.